dev/com.ibm.ws.security.social/resources/com/ibm/ws/security/social/resources/SocialMessages.nlsprops

```diff
@@ -119,6 +119,16 @@ JSON_ENTRY_WRONG_JSON_TYPE=CWWKS5386E: The value for the [{0}] key in the JSON d
 JSON_ENTRY_WRONG_JSON_TYPE.explanation=The value for the specified key did not have the correct type. The JSON data might be malformed or might have an unexpected structure.
 JSON_ENTRY_WRONG_JSON_TYPE.useraction=Check the structure of the JSON data. Check the JSON data to see whether an error occurred.
 
+# 0=Exception message
+INTROSPECT_ERROR_GETTING_USER_INFO=CWWKS5387E: The response from the token introspection API cannot be processed: {0}
+INTROSPECT_ERROR_GETTING_USER_INFO.explanation=Information about the authenticated user cannot be obtained because another error occurred.
+INTROSPECT_ERROR_GETTING_USER_INFO.useraction=If the message contains another error message, see the user action for that message. Otherwise, check the server logs for more error messages that might indicate where the other error occurred.
+
+# 0=User API response string
+INTROSPECT_USER_API_INACTIVE=CWWKS5388E: The token introspection request failed because the token that is included in the request is not valid. The full response is {0}.
+INTROSPECT_USER_API_INACTIVE.explanation=Either the authorization server did not issue the token, the resource owner revoked the token, the token is expired, or another error occurred.
```

Comment:
Suggestion: remove the "Either." Change "Either the" to "The" because either typically is used when there are only 2 options.